oo-install/atomic-openshift-utils.spec

```diff
@@ -0,0 +1,58 @@
+%{!?python_sitelib: %global python_sitelib %(%{__python} -c "from distutils.sysconfig import get_python_lib; print get_python_lib()")}
+
+%global import_path github.com/openshift/openshift-extras
+# %commit is intended to be set by tito custom builders provided
+# in the .tito/lib directory. The values in this spec file will not be kept up to date.
+%{!?commit:
+%global commit c64d09e528ca433832c6b6e6f5c7734a9cc8ee6f
+}
+
+Name:           atomic-openshift-utils
+Version:        3.0.0
```

Comment:
As we iterate on builds with this, is it ok to bump this to 3.0.1, 3.0.2, etc? Given we're on a 3.1 release I can't imagine we're keeping parity with anything important.